controllers/admin/customerController.js

```diff
@@ -39,14 +39,9 @@ exports.adminLogin = async (req, res, next) => {
         throw error;
       }
 
-      if (user.role === "customer") {
-        const error = new Error("Invalid credentials entered!");
-        error.statusCode = 500;
-        throw error;
-      }
-
       var userObj = {
         userId: user._id,
+        email:user.email,
         phoneNumber: user.phoneNumber,
```

Comment:
phonenumber not used in this.so remove from this

Reply:
removed it